sdk/src/test/java/me/uport/sdk/fakes/InMemoryAccountStorage.kt

```diff
@@ -8,6 +8,14 @@ import me.uport.sdk.identity.Account
  */
 class InMemoryAccountStorage : AccountStorage {
```

Comment:
This class does not seem to be in use any more.
I think it's safe to delete entirely.

Reply:
ok